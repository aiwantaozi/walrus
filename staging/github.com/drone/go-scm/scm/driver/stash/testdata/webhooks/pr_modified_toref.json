{
    "eventKey": "pr:modified",
    "date": "2021-11-23T13:16:20+0000",
    "actor": {
        "name": "jcitizen",
        "emailAddress": "jane@example.com",
        "id": 1,
        "displayName": "Jane Citizen",
        "active": true,
        "slug": "jcitizen",
        "type": "NORMAL",
        "links": {
            "self": [
                {
                    "href": "http://example.com:7990/users/jcitizen"
                }
            ]
        }
    },
    "pullRequest": {
        "id": 2,
        "version": 0,
        "title": "added LICENSE",
        "description": "added BSD license text",
        "state": "OPEN",
        "open": true,
        "closed": false,
        "createdDate": 1637591800463,
        "updatedDate": 1637673380977,
        "fromRef": {
            "id": "refs/heads/feature-2",
            "displayId": "feature-2",
            "latestCommit": "c00d0f269a9844dd168f72f6a9d78a9cff49a318",
            "type": "BRANCH",
            "repository": {
                "slug": "my-repo",
                "id": 1,
                "name": "my-repo",
                "hierarchyId": "5aa23c1a1280b0c0acce",
                "scmId": "git",
                "state": "AVAILABLE",
                "statusMessage": "Available",
                "forkable": true,
                "project": {
                    "key": "PRJ",
                    "id": 2,
                    "name": "PRJ",
                    "public": false,
                    "type": "NORMAL",
                    "links": {
                        "self": [
                            {
                                "href": "http://example.com:7990/projects/PRJ"
                            }
                        ]
                    }
                },
                "public": false,
                "links": {
                    "clone": [
                        {
                            "href": "http://example.com:7990/scm/prj/my-repo.git",
                            "name": "http"
                        },
                        {
                            "href": "ssh://git@example.com:7999/prj/my-repo.git",
                            "name": "ssh"
                        }
                    ],
                    "self": [
                        {
                            "href": "http://example.com:7990/projects/PRJ/repos/my-repo/browse"
                        }
                    ]
                }
            }
        },
        "toRef": {
            "id": "refs/heads/master",
            "displayId": "master",
            "latestCommit": "0667def9e377a8fa76ce67b6d4e53e21b246f030",
            "type": "BRANCH",
            "repository": {
                "slug": "my-repo",
                "id": 1,
                "name": "my-repo",
                "hierarchyId": "5aa23c1a1280b0c0acce",
                "scmId": "git",
                "state": "AVAILABLE",
                "statusMessage": "Available",
                "forkable": true,
                "project": {
                    "key": "PRJ",
                    "id": 2,
                    "name": "PRJ",
                    "public": false,
                    "type": "NORMAL",
                    "links": {
                        "self": [
                            {
                                "href": "http://example.com:7990/projects/PRJ"
                            }
                        ]
                    }
                },
                "public": false,
                "links": {
                    "clone": [
                        {
                            "href": "http://example.com:7990/scm/prj/my-repo.git",
                            "name": "http"
                        },
                        {
                            "href": "ssh://git@example.com:7999/prj/my-repo.git",
                            "name": "ssh"
                        }
                    ],
                    "self": [
                        {
                            "href": "http://example.com:7990/projects/PRJ/repos/my-repo/browse"
                        }
                    ]
                }
            }
        },
        "locked": false,
        "author": {
            "user": {
                "name": "jcitizen",
                "emailAddress": "jane@example.com",
                "id": 1,
                "displayName": "Jane Citizen",
                "active": true,
                "slug": "jcitizen",
                "type": "NORMAL",
                "links": {
                    "self": [
                        {
                            "href": "http://example.com:7990/users/jcitizen"
                        }
                    ]
                }
            },
            "role": "AUTHOR",
            "approved": false,
            "status": "UNAPPROVED"
        },
        "reviewers": [],
        "participants": [],
        "links": {
            "self": [
                {
                    "href": "http://example.com:7990/projects/PRJ/repos/my-repo/pull-requests/2"
                }
            ]
        }
    },
    "previousTitle": "old title",
    "previousDescription": "old description",
    "previousTarget": {
        "id": "refs/heads/wrong",
        "displayId": "wrong",
        "type": "BRANCH",
        "latestCommit": "2bbad782fd5453ded95fc5ce8b16839ff10615e0",
        "latestChangeset": "2bbad782fd5453ded95fc5ce8b16839ff10615e0"
    }
}
