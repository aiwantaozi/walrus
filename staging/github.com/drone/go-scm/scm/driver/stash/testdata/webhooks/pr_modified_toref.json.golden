{
    "Action": "synchronized",
    "Repo": {
        "ID": "1",
        "Namespace": "PRJ",
        "Name": "my-repo",
        "Perm": null,
        "Branch": "master",
        "Private": true,
        "Clone": "http://example.com:7990/scm/prj/my-repo.git",
        "CloneSSH": "ssh://git@example.com:7999/prj/my-repo.git",
        "Link": "http://example.com:7990/projects/PRJ/repos/my-repo/browse",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
    },
    "PullRequest": {
        "Number": 2,
        "Title": "added LICENSE",
        "Body": "added BSD license text",
        "Sha": "c00d0f269a9844dd168f72f6a9d78a9cff49a318",
        "Ref": "refs/pull-requests/2/from",
        "Source": "feature-2",
        "Target": "master",
        "Fork": "PRJ/my-repo",
        "Link": "http://example.com:7990/projects/PRJ/repos/my-repo/pull-requests/2",
        "Closed": false,
        "Merged": false,
        "Base": {
            "Name": "master",
            "Path": "refs/heads/master",
            "Sha": "0667def9e377a8fa76ce67b6d4e53e21b246f030"
        },
        "Head": {
            "Name": "feature-2",
            "Path": "refs/heads/feature-2",
            "Sha": "c00d0f269a9844dd168f72f6a9d78a9cff49a318"
        },
        "Author": {
            "Login": "jcitizen",
            "Name": "Jane Citizen",
            "Email": "jane@example.com",
            "Avatar": "https://www.gravatar.com/avatar/9e26471d35a78862c17e467d87cddedf.jpg"
        },
        "Created": "2021-11-22T14:36:40Z",
        "Updated": "2021-11-23T13:16:20Z"
    },
    "Sender": {
        "Login": "jcitizen",
        "Name": "Jane Citizen",
        "Email": "jane@example.com",
        "Avatar": "https://www.gravatar.com/avatar/9e26471d35a78862c17e467d87cddedf.jpg"
    }
}
